plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -446,6 +449,29 @@ private void prepareHost(HostMO hostMo, String privateTrafficLabel) throws Excep
         }
     }
 
+    private HostMO getOldestExistentHostInCluster(Long clusterId, VmwareContext serviceContext) throws Exception {
+        HostVO host = hostDao.findOldestExistentHypervisorHostInCluster(clusterId);
+        if (host == null) {
+            return null;
+        }
+
+        ManagedObjectReference morSrcHost = HypervisorHostHelper.getHypervisorHostMorFromGuid(host.getGuid());
+        if (morSrcHost == null) {
+            Map<String, String> clusterDetails = clusterDetailsDao.findDetails(clusterId);
+            if (clusterDetails.get("url") == null) {
```

Comment:
Can we use `StringUtils.isBlank(..)` here, since cluster_details does not accept null values anymore?

Reply:
> Can we use `StringUtils.isBlank(..)` here, since cluster_details does not accept null values anymore?

done